test/v1/validation/AccountingPeriodValidatorSpec.scala

```diff
@@ -72,14 +89,14 @@ class AccountingPeriodValidatorSpec extends BaseSpec {
         )
         leftSideError(model.validate).errorMessage shouldBe AccountingPeriod18MonthsMax(endDate).errorMessage
       }
-      "End date is in the future" in {
 
-        val endDate = LocalDate.now().plusDays(1)
+      "Start date is before minimum" in {
+        val startDate = minimumStartDate.minusDays(1L)
         val model = accountingPeriodModel.copy(
-          startDate = LocalDate.now().minusMonths(12),
-          endDate = endDate
+          startDate = startDate,
```

Comment:
Is this spacing correct? It might just be the viewer being weird for me.

Reply:
What looks wrong with it?